crates/ruma-events/src/key/verification/start.rs

```diff
@@ -489,6 +558,41 @@ mod tests {
                 && method == "m.sas.custom"
                 && data.get("test").unwrap() == &JsonValue::from("field")
         );
+
+        #[cfg(feature = "unstable-pre-spec")]
+        {
+            let json = json!({
+                "content": {
+                    "from_device": "123",
+                    "method": "m.reciprocate.v1",
+                    "secret": "It's a secret to everybody",
+                    "transaction_id": "456",
+                },
+                "type": "m.key.verification.start",
+                "sender": sender,
+            });
+
+            assert_matches!(
+                from_json_value::<Raw<ToDeviceEvent<StartToDeviceEventContent>>>(json)
+                    .unwrap()
+                    .deserialize()
+                    .unwrap(),
+                ToDeviceEvent {
+                    sender,
+                    content: StartToDeviceEventContent {
+                        from_device,
+                        transaction_id,
+                        method: StartMethod::MReciprocateV1(
+                            MReciprocateV1Content {
+                                secret
+                        }),
```

Comment:
The formatting here is a bit weird. The closing curly brace should line up in indentation with the line that has the opening curly brace, if there are extra linebreaks here at all. Though I think everything on one line is okay here:

```suggestion
                        method: StartMethod::MReciprocateV1(MReciprocateV1Content { secret }),
```

same below.